activemodel/lib/active_model/errors.rb

```diff
@@ -364,12 +364,49 @@ def full_messages_for(attribute)
     # Returns a full message for a given attribute.
     #
     #   person.errors.full_message(:name, 'is invalid') # => "Name is invalid"
+    #
+    #   The `"%{attribute} %{message}"` error format can be overridden either
```

Comment:
Can we document this in the same way as https://github.com/rails/rails/blob/9f95767979579f5761cb0d2bcccb67f3662349c5/activemodel/lib/active_model/errors.rb#L393-L400?